src/BarrelCalorimeterScFi_geo.cpp

```diff
@@ -25,19 +25,20 @@ using namespace dd4hep;
 typedef ROOT::Math::XYPoint Point;
```

Comment:
All these changes are just pre-commit clang-format changes caused by the change from module to sector.